src/contour/Config.h

```diff
@@ -445,7 +445,7 @@ struct TerminalProfile
     ConfigEntry<HyperlinkDecorationConfig, documentation::HyperlinkDecoration> hyperlinkDecoration {};
 
     ConfigEntry<std::string, documentation::WMClass> wmClass { CONTOUR_APP_ID };
-    ConfigEntry<bool, documentation::OptionKeyAsAlt> optionKeyAsAlt { false };
+    ConfigEntry<bool, documentation::CommandKeyAsAlt> commandKeyAsAlt { true };
```

Comment:
do we really want to default to `true` here?

Reply:
I think that this is better for the default since user do not need to tweak config to use Alt+something key mappings on mac